orangecontrib/text/corpus.py

```diff
@@ -230,8 +229,6 @@ def copy(self):
         """Return a copy of the table."""
         c = self.__class__(self.X, self.Y, self.metas, self.domain, self.text_features)
         c.ensure_copy()
-        c._tokens = copy.copy(self._tokens)
-        c._dictionary = copy.copy(self._dictionary)
```

Comment:
Deleting this would also remove preprocessing when making a copy. I believe what we need is something like:
```
c._tokens = self._tokens
c._dictionary = self._dictionary
```